requirements/test.txt

```diff
@@ -1,8 +1,8 @@
 coverage>=5.0
 codecov>=2.1
 pytest>=5.0
-pytest-cov
-pytest-flake8
+# pytest-cov
```

Comment:
Any particular reason why these are commented out?

Reply:
I did it now bc Mac was complaining and we do need it anymore